### ISSUE TITLE
jquery resolved version is wrong

### ISSUE THREAD

**Issue by FDiskas** [yarnpkg/yarn#1636]
**Do you want to request a *feature* or report a *bug*?**
BUG

**What is the current behavior?**
BUG

**If the current behavior is a bug, please provide the steps to reproduce.**
https://gist.github.com/FDiskas/43e2c116069306d7b0aace9ade628675

**What is the expected behavior?**
There should nod be installed `jquery` as submodule of `bootstrap-datepicker`

**Please mention your node.js, yarn and operating system version.**
```
$ node --version
v6.9.1
```
```
$ yarn --version
0.16.1
```
```
$ npm --version
3.8.0
```
Operating system:
```
OS: Fedora 24
Procesor: Intel® Core™ i5-4210M CPU @ 2.60GHz × 4 
OS Type: 64-bit
```

**Comment by FDiskas** [comment]
As a workaround I should install latest jQuery to solve this issue.

**Comment by kittens** [comment]
This was fixed by #1477.
